### PR TITLE
i18n(fr): Adding `integrations-guide.md` translations

### PR DESCRIPTION
**Pull request by Chrissdroid**
Adding `guides/integrations-guide.md` into the french translation.

Open and ready for review !

### CODE REVIEW

**Comment by sarah11918** [comment]
Just updating the branch, and pinging because this one's been sitting here a while! Anyone free to review?  @Olyno ? :)

File: src/pages/fr/guides/integrations-guide.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+- Ajoute de nouvelles fonctionnalités à votre projet, comme la génération automatique du sitemap.
+- Écrivez du code personnalisé qui s'intégre dans le processus de construction, le serveur de développement et plus encore.
+
+> Les intégrations sont toujours en cours de développement, et l'API n'a pas encore été finalisée. Seules les intégrations officielles (celles publiées sur `@astrojs/` sur npm) sont actuellement prises en charge pour protéger les utilisateurs des changements brusques.
+>
+> **Pour activer les intégrations tierces :** Exécutez Astro avec la commande CLI `--experimental-integrations`.
+
+## Tutoriel : Ajouter React à votre projet
+
+Dans cet exemple, nous allons installer l'intégration `@astrojs/react` pour ajouter le support React à votre projet Astro. Le processus pour ajouter un autre Framework (Preact, Vue, Svelte ou Solid.js) est presque identique et peut être suivi en utilisant les mêmes étapes que ci-dessous.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:21]:
```suggestion
Dans cet exemple, nous allons ajouter l'intégration `@astrojs/react` pour ajouter le React à votre projet Astro. Le processus d'ajout de tout autre framework (Preact, Vue, Svelte ou Solid) est presque identique et peut être suivi en utilisant les mêmes étapes décrites ci-dessous.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+<blockquote>
+  <Badge variant="accent">Sentez-vous l'appel de l'aventure ?</Badge>
+
+  Astro a récemment lancé une commande **expérimentale** `astro add` pour automatiser ce processus ! Au lieu des étapes ci-dessous, vous pouvez exécuter `npx astro add react`. C'est tout !
+
+  Passez à [Configuration d'intégration automatique](#configuration-dintégration-automatique) pour plus de détails.
+</blockquote>
+
+Premièrement, vous devez installer l'intégration et les packages liés que vous attendez d'utiliser dans votre projet. Pour React, cela signifie installer l'intégration `@astrojs/react` ***et*** les packages `react` + `react-dom`.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:31]:
```suggestion
Tout d'abord, vous devrez installer à la fois l'intégration et tous les modules liés que vous pensez utiliser dans votre projet. Pour React, cela signifie installer l'intégration `@astrojs/react` ****et*** les modules `react` + `react-dom`.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+```bash
+npm install --save-dev @astrojs/react
+```
+
+Une fois que vos packages ont été installés, ajoutez deux lignes au fichier de configuration de votre projet `astro.config.mjs`.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:37]:
```suggestion
Une fois que vos modules ont été installés, ajoutez deux nouvelles lignes au fichier de configuration de votre projet `astro.config.mjs`.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import react from '@astrojs/react';
+
+  export default defineConfig({
++   integrations: [react()],
+  });
+```
+
+La première ligne est l'importation l'intégration dans votre fichier de configuration. La deuxième ligne appelle la fonction d'intégration (`react()`) et ajoute l'intégration pour que Astro sache qu'il doit l'utiliser.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:49]:
```suggestion
La première ligne est l'importation de l'intégration dans votre fichier de configuration. La deuxième ligne appelle la fonction d'intégration (`react()`) et ajoute l'intégration pour qu'Astro sache l'utiliser.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+C'est tout ! Redémarrez Astro, et la nouvelle intégration devrait prendre effet immédiatement.
+
+Si vous voyez une erreur au démarrage, assurez-vous que vous avez :
+
+- ✅ ajouté les packages requis avec npm
+- ✅ importé l'intégration dans votre fichier `astro.config.mjs`
+- ✅ appelé votre intégration comme une fonction (`[react()]`, pas `[react]`)
+- ✅ supprimé toute configuration obsolète `renderers:` (avant la version 0.25)
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:58]:
```suggestion
Si vous voyez une erreur au démarrage, assurez-vous que vous avez :

- ✅ installé les modules requis avec npm
- ✅ importé l'intégration dans votre fichier `astro.config.mjs`
- ✅ appelé votre intégration comme une fonction (`[react()]`, pas `[react]`)
- ✅ supprimé toute configuration `renderers:` obsolète (avant la version 0.25)
```

File: src/pages/fr/guides/integrations-guide.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+## Configuration d'intégration automatique
+
+Astro a récemment lancé une commande **expérimentale** `astro add` pour automatiser la configuration des intégrations.
+
+> Nous allons toujours demander une confirmation avant de mettre à jour tous vos fichiers, mais il n'est jamais trop prudent de faire un backup de version juste pour être sûr.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:64]:
```suggestion
> Nous vous demanderons toujours une confirmation avant de mettre à jour l'un de vos fichiers, mais il n'est jamais inutile de disposer d'une sauvegarde, au cas où.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+À la place de la configuration manuelle décrite ci-dessus, exécutez simplement `astro add [name]` et notre assistant d'intégration automatique mettra à jour votre fichier de configuration et installera toutes les dépendances nécessaires.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:66]:
```suggestion
Au lieu de la configuration manuelle décrite ci-dessus, exécutez simplement `astro add [name]` et notre assistant d'intégration automatique mettra à jour votre fichier de configuration et installera toutes les dépendances nécessaires.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+npx astro add react tailwind partytown
+# Utilisant Yarn
+yarn astro add react tailwind partytown
+# Utilisant PNPM
+pnpx astro add react tailwind partytown
+```
+
+## Gérer les dépendances d'intégration
+
+Quand vous installez une intégration Astro dans votre projet, gardez une oeil sur les avertissements `"missing peer dependencies"` que vous voyez pendant l'installation. Tous les gestionnaires de paquets ne peuvent pas vous les installer automatiquement. Si vous êtes sur Node v16+ et utilisez npm, vous n'aurez pas besoin de vous inquiéter de cette section.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:90]:
```suggestion
Lorsque vous installez une intégration Astro dans votre projet, soyez attentif aux avertissements `"missing peer dependencies"` qui apparaissent au cours de l'étape d'installation. Tous les gestionnaires de modules n'installent pas automatiquement les dépendances pour vous. Si vous utilisez Node v16+ et npm, vous n'avez pas à vous soucier de cette section.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+
+Si vous voyez un avertissement `"Cannot find package 'react'"` (ou similaire), cela signifie que vous devez installer ce package dans votre projet. React, par exemple, est une dépendance de la paquetage `@astrojs/react`. Cela signifie que vous devez installer les paquets officiels `react` et `react-dom` avec votre intégration. L'intégration va alors utiliser ces packages automatiquement.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:92]:
```suggestion
Si vous voyez un avertissement `"Cannot find package 'react'"` (ou un avertissement similaire) lorsque vous démarrez Astro, cela signifie que vous devez installer ce module dans votre projet. React, par exemple, est une dépendance de l'intégration `@astrojs/react`. Cela signifie que vous devez installer les modules officiels `react` et `react-dom` en même temps que votre intégration. L'intégration puisera automatiquement dans ces modules.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+
+```diff
+# Exemple : Installer les intégrations et les Frameworks ensemble
+- npm install @astrojs/react
++ npm install @astrojs/react react react-dom
+```
+
+Si vous manquez cette étape, ne vous inquiétez pas, Astro vous avertira au démarrage si des dépendances en commun manquent mais ne sont pas dans votre projet.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:100]:
```suggestion
Si vous manquez cette étape, ne vous inquiétez pas, Astro vous avertira au cours du démarrage si des dépendances peer manquantes sont requises mais non trouvées dans votre projet.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+
+Gérer vos propres dépendances en commun peut être un peu plus de travail, mais il vous permet de contrôler précisément quelles versions de paquets vous utilisez pour les choses comme React, Tailwind, et plus encore. Cela vous donne plus de contrôle sur votre projet.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:102]:
```suggestion
La gestion de vos propres dépendances peer peut représenter un peu plus de travail, mais elle vous permet également de contrôler exactement les versions des modules que vous utilisez pour des choses comme React, Tailwind, et plus encore. Cela vous donne plus de contrôle sur votre projet.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
+
+Dans le futur, une commande utilitaire `astro add` pourrait gérer tout ce processus pour vous, et installer les dépendances correspondantes pour vos intégrations automatiquement.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:104]:
```suggestion
À l'avenir, une commande utile `astro add` sera capable de gérer toute cette configuration pour vous, et d'installer automatiquement les dépendances correctes des peers pour vos intégrations.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+
+## Utiliser les intégrations
+
+Les intégrations Astro sont toujours ajoutées à l'aide de la propriété `integrations` dans votre fichier `astro.config.mjs`.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:108]:
```suggestion
Les intégrations Astro sont toujours ajoutées par le biais de la propriété `integrations` dans votre fichier `astro.config.mjs`.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 14]
```diff
@@ -0,0 +1,164 @@
...
+
+> Vous voulez en savoir plus sur l'utilisation ou la configuration d'une intégration spécifique ? Consultez notre [répertoire des intégrations](https://astro.build/integrations/) et suivez le lien vers son dépôt sur GitHub.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:110]:
```suggestion
> Vous souhaitez en savoir plus sur l'utilisation ou la configuration d'une intégration spécifique ? Trouvez-la dans notre [répertoire des intégrations](https://astro.build/integrations/) et suivez le lien vers son dépôt sur GitHub.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+
+Il existe trois façons d'importer une intégration dans votre projet Astro :
+
+1. Installer une intégration de paquets npm.
+2. Importer votre propre intégration à partir d'un fichier local dans votre projet.
+3. Écrire votre intégration en ligne, directement dans votre fichier de configuration.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:116]:
```suggestion
Il existe trois façons courantes d'importer une intégration dans votre projet Astro :

1. En installant une intégration à partir d'un module npm.
2. En important votre propre intégration à partir d'un fichier local dans votre projet.
3. En écrivant votre intégration en ligne, directement dans votre fichier de configuration.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+    installedIntegration(),
+    // 2. Importée à partir d'un fichier JS local
+    localIntegration(),
+    // 3. Un objet directement dans le fichier de configuration
+    {name: 'namespace:id', hooks: { /* ... */ }},
+  ]
+})
+```
+
+Consultez l'[API d'intégration](/fr/reference/integrations-reference/) pour en savoir plus sur les différentes façons que vous pouvez écrire une intégration.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:136]:
```suggestion
Consultez la référence à l'[API d'intégration](/fr/reference/integrations-reference/) pour connaître toutes les différentes façons d'écrire une intégration.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+
+### Options personnalisées
+
+Les intégrations sont presque toujours écrites en tant que fonctions de fabrique qui renvoient l'objet intégration. Cela vous permet de passer des arguments et des options à la fonction de fabrique qui personnalise l'intégration pour votre projet.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:140]:
```suggestion
Les intégrations sont presque toujours conçues comme des fonctions "factory" qui renvoient l'objet d'intégration. Cela vous permet de passer des arguments et des options à la fonction afin de personnaliser l'intégration pour votre projet.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 18]
```diff
@@ -0,0 +1,164 @@
...
+```js
+integrations: [
+  // Exemple : Personnalisez votre intégration avec des arguments de fonction
+  sitemap({filter: true})
+]
+```
+
+### Activer une intégration
+
+Les intégrations `false` sont ignorées, donc vous pouvez activer et désactiver les intégrations sans vous inquiéter de valeurs booléennes ou `undefined` qui restent.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:151]:
```suggestion
Les intégrations `false` sont ignorées, ce qui vous permet d'activer et de désactiver les intégrations sans vous soucier des valeurs booléennes et des `undefined` restants.
```

File: src/pages/fr/guides/integrations-guide.md [hunk 19]
```diff
@@ -0,0 +1,164 @@
...
+  // Exemple : Ignorer la construction d'un sitemap sur Windows
+  process.platform !== 'win32' && sitemap()
+]
+```
+
+## Construire votre propre intégration
+
+L'API d'intégration d'Astro est inspirée de Rollup et Vite, et est conçue pour ressembler à celles-ci et aider ceux qui sont familier avec cet environnement.
+
+Consultez l'[API d'intégration](/fr/reference/integrations-reference/) pour savoir ce que peut faire une intégration et comment la créer vous-même.
```
Comment (Olyno) [inline src/pages/fr/guides/integrations-guide.md:164]:
```suggestion
L'API d'intégration d'Astro est inspirée de Rollup et de Vite, et est conçue pour être familière à toute personne ayant déjà écrit un plugin Rollup ou Vite.

Consultez l'[API d'intégration](/fr/reference/integrations-reference/) pour savoir ce que les intégrations peuvent faire et comment en écrire une vous-même.
```

**Review by Olyno — CHANGES_REQUESTED** [review]

**Comment by sarah11918** [comment]
Merging, because I see all the edits resolved! If you have issues, reopen with another PR. But, The French-Brazilian Word War must go on!

**Pull request merged**